src/WebApiContrib.Formatting.MsgPack/MessagePackMediaTypeFormatter.cs

```diff
@@ -14,7 +18,7 @@ namespace WebApiContrib.Formatting.MsgPack
     /// <remarks>Converted from Filip W.'s MessagePackMediaTypeFormatter.</remarks>
     /// <see href="http://msgpack.org/"/>
     /// <see href="http://www.strathweb.com/2012/09/boost-up-your-asp-net-web-api-with-messagepack/"/>
-    public class MessagePackMediaTypeFormatter : BufferedMediaTypeFormatter
+    public class MessagePackMediaTypeFormatter : MediaTypeFormatter
```

Comment:
Personal preference rather than a compelling reason I'm afraid. Given that reasoning I'll switch it back if you feel strongly about it.